engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/number/Decimal.java

```diff
@@ -3,8 +3,8 @@
 import org.enso.interpreter.dsl.BuiltinType;
 import org.enso.interpreter.node.expression.builtin.Builtin;
 
-@BuiltinType(name = "Standard.Base.Data.Numbers.Decimal")
-public class Decimal extends Builtin {
+@BuiltinType(name = "Standard.Base.Data.Numbers.Float")
+public class Float extends Builtin {
```

Comment:
This change is not going to compile. You cannot have a `Float` class in `Decimal.java` code.